tests/Basic-Auth-master/basic-auth.php

```diff
@@ -0,0 +1,62 @@
+<?php
```

Comment:
I'm not sure if we need this plugin in the test environment, as I don't see its usage anywhere in tests. Could you please remove it if are not using it? Thanks